src/main/java/uk/gov/hmcts/reform/sscs/factory/CcdNotificationWrapper.java

```diff
@@ -137,13 +137,13 @@ public SscsCaseData getOldSscsCaseData() {
                 || TCW_DECISION_APPEAL_TO_PROCEED.equals(getNotificationType())
                 || NON_COMPLIANT_NOTIFICATION.equals(getNotificationType())
                 || RESEND_APPEAL_CREATED_NOTIFICATION.equals(getNotificationType())
-                || REQUEST_INFO_INCOMPLETE.equals(getNotificationType())
-                || JOINT_PARTY_ADDED.equals(getNotificationType()))
+                || JOINT_PARTY_ADDED.equals(getNotificationType())
+                || isValidRequestInfoIncompleteEventForParty(PartyItemList.APPELLANT))
         ) {
             subscriptionWithTypeList.add(new SubscriptionWithType(getAppointeeSubscription(), APPOINTEE));
-        } else if (getOldSscsCaseData() != null && isValidReviewConfidentialityRequest(getOldSscsCaseData().getConfidentialityRequestOutcomeAppellant(), getNewSscsCaseData().getConfidentialityRequestOutcomeAppellant())) {
-            subscriptionWithTypeList.add(new SubscriptionWithType(getAppellantSubscription(), APPELLANT));
-        } else if (!REVIEW_CONFIDENTIALITY_REQUEST.equals(getNotificationType())) {
+        } else if (getOldSscsCaseData() != null && isValidReviewConfidentialityRequest(getOldSscsCaseData().getConfidentialityRequestOutcomeAppellant(), getNewSscsCaseData().getConfidentialityRequestOutcomeAppellant())
+                || isValidRequestInfoIncompleteEventForParty(PartyItemList.APPELLANT)
+                || !notificationsMaybeNotForAppellant.contains(getNotificationType())) {
```

Comment:
This does not look correct. Would this condition let in notifications to be sent to the appellant that previously were not allowed?

Reply:
Spoke to Ryan and determined the logic here is correct, but a little confusing. I've split the methods up now and added a comment to the appellant one to try and make it clearer.